.github/workflows/new-brassy-note.yaml

```diff
@@ -0,0 +1,92 @@
+name: Generate Release Note with Brassy
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
```

Comment:
Oh, we probably explicitly only want this to run on tag/release, because otherwise the most recent tag will be the previous version, so we will be overwriting (?) the previous versions' release notes.

"update_this_release_note" would tell you the correct release note file to write to in between releases - both for every PR merge, as well as immediately after tag/release.  I tag/release the current version on the repository BEFORE updating "update_this_release_note" - the first thing merged in after a version release is the new "update_this_release_note", so it should always be accurate.

If we run this on every PR merge, then the plugin repositories would have to clone the geoips repo in this CI in order to check what "update_this_release_note" is, to see which release note to create.